app/file.go

```diff
@@ -462,7 +461,7 @@ func prepareImage(fileData []byte) (*image.Image, int, int) {
 	}
 
 	// Flip the image to be upright
-	orientation, _ := getImageOrientation(bytes.NewReader(fileData))
+	orientation, _ := getImageOrientation(fileData)
```

Comment:
With `fileData` having been read above, this is now empty and `getImageOrientation` won't do anything.

Reply:
This is a bug that "forgets" rewinds before use. Thank you for your correction.